src/OpenTelemetry.Contrib.Instrumentation.AWSLambda/TracerProviderBuilderExtensions.cs

```diff
@@ -0,0 +1,49 @@
+﻿// <copyright file="TracerProviderBuilderExtensions.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using OpenTelemetry.Contrib.Instrumentation.AWSLambda.Implementation;
+using OpenTelemetry.Resources;
+
+namespace OpenTelemetry.Trace
+{
+    /// <summary>
+    /// Extension class for TracerProviderBuilder.
+    /// </summary>
+    public static class TracerProviderBuilderExtensions
+    {
+        /// <summary>
+        /// Add AWS Lambda configurations.
+        /// </summary>
+        /// <param name="builder"><see cref="TracerProviderBuilder"/> being configured.</param>
+        /// <returns>The instance of <see cref="TracerProviderBuilder"/> to chain the calls.</returns>
+        public static TracerProviderBuilder AddLambdaConfigurations(this TracerProviderBuilder builder)
+        {
+            if (builder == null)
+            {
+                throw new ArgumentNullException(nameof(builder));
+            }
+
+            builder.AddSource(AWSLambdaUtils.ActivitySourceName);
+            builder.SetResourceBuilder(ResourceBuilder
+                .CreateEmpty()
```

Comment:
Instead of an empty resource, can we use telemetry sdk resource?  It comes with attributes about the OTel SDK.
https://github.com/open-telemetry/opentelemetry-dotnet/blob/main/src/OpenTelemetry/Resources/ResourceBuilderExtensions.cs#L97

Reply:
I'm guessing you're asking to add the telemetry sdk **after** we create the `ResourceBuilder` as there are only two options: `CreateEmpty()` and `CreateDefault()`, which makes no difference in Lambda as what `CreateDefault()` added will also be added by our resource detector using `CreateEmpty()`. And I didn't see we need to include the telemetry sdk resource attribute for a Lambda span in the [specification](https://github.com/open-telemetry/opentelemetry-specification/blob/86aeab1e0a7e6c67be09c7f15ff25063ee6d2b5c/specification/trace/semantic_conventions/instrumentation/aws-lambda.md#all-triggers) so I need to confirm, but IMO, I agree it's good to include.

Reply:
Yeah. Can be done as a follow up.